apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -219,5 +219,5 @@
 
     public static final OfficialComponent APACHE_KYLIN_JDBC_DRIVER = new OfficialComponent(121, "apache-kylin-jdbc-driver");
 
-    public static final OfficialComponent GUAVA_EVENT_BUS = new OfficialComponent(122, "GuavaEventBus");
+    public static final OfficialComponent GUAVA_EVENT_BUS = new OfficialComponent(123, "GuavaEventBus");
```

Comment:
Please register this to the main repo first.

Reply:
OK, I just submit a PR to main repo.

Reply:
https://github.com/apache/skywalking/pull/8524